doc/api/next_api_changes/deprecations/21992-AL.rst

```diff
@@ -0,0 +1,8 @@
+``lastrect``
+~~~~~~~~~~~~
+The ``lastrect`` attribute of ``NavigationToolbar2Tk`` and ``RubberbandTk`` is
+deprecated.
+
+``ConfigureSubplotGTK``
+~~~~~~~~~~~~~~~~~~~~~~~
+This class no longer inherits from ``Gtk.Window``.
```

Comment:
That's not a deprecation.

Reply:
Do we have to mention this at all? The module the class is in is private.

Reply:
Moved it to behavior/, but actually, indeed, the class is completely private.

Reply:
If the class is private we should bother users with a notification at all.

Reply:
removed.